prow/config.yaml

```diff
@@ -15648,14 +15648,14 @@ periodics:
   spec:
     containers:
     - args:
+      - --bare
```

Comment:
does this work? don't we need to check out `test-infra`? or are we piggy-backing on the fact that `test-infra` is already checked out for bootstrap?